elastic/beats
# Disable monitoring during fleetserver bootstrap process

## What does this PR do?

Set the monitoring.enabled flag to false during the fleet-server
bootstrap process in order to ensure that no data is sent to an
elasticsearch instance running on the same host.

## Why is it important?

In a self-hosted scenario fleet will show that some data-streams have stopped in certain cases (such as after the fleet-server restarts). However these datastreams are incorrectly sent by the agent during the bootstrapping processes when the default credentials for output match Elasticsearch's credentials.

## Checklist

- [] ~~I have commented my code, particularly in hard-to-understand areas~~
- [] ~~I have made corresponding changes to the documentation~~
- [] ~~I have made corresponding change to the default configuration files~~
- [] ~~I have added tests that prove my fix is effective or that my feature works~~

## How to test this PR locally

Run ES + fleet locally (`elastic-package` can do this), view the datastreams tab in the fleet-ui, the `elastic_agent.metricsbeat` and `elastic_agent.filebeat` streams should not generate.

## Related issues

- Closes https://github.com/elastic/fleet-server/issues/376
